web/src/features/AppVersionHistory/api/getVersions.jsx

```diff
@@ -0,0 +1,190 @@
+import { useQuery } from "react-query";
+import { Utilities } from "../../../utilities/utilities";
+import { useParams } from "react-router-dom";
+import { useCurrentApp } from "../hooks/useCurrentApp";
+import { useMetadata } from "@src/stores";
+import { useIsHelmManaged } from "@src/components/hooks";
+
+async function getVersions({
+  accessToken = Utilities.getToken(),
+  apiEndpoint = process.env.API_ENDPOINT,
+  _fetch = fetch,
+  currentPage = 0,
+  pageSize = 20,
+  slug,
+} = {}) {
+  try {
+    const res = await _fetch(
+      `${apiEndpoint}/app/${slug}/versions?currentPage=${currentPage}&pageSize=${pageSize}&pinLatestDeployable=true`,
+      {
+        headers: {
+          Authorization: accessToken,
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      }
+    );
+    if (!res.ok) {
+      if (res.status === 401) {
+        Utilities.logoutUser();
+        return null;
+      }
+      throw Error(`Failed to fetch apps with status ${res.status}`);
+    }
+    return await res.json();
+  } catch (err) {
+    throw Error(err);
+  }
+}
+
+// TODO: refactor this function so that the airgapped / nonairgapped are separate
```

Comment:
I opted to just leave this the same for non helm managed stuff.  Right now I'm thinking of kots in terms of the types of installers its using:
- airgapped 
- kots managed
- helm managed 

The UI seems to diverge along these pretty cleanly so we should figure out how to handle label / behavior changes.